src/test/java/org/springframework/cloud/deployer/spi/kubernetes/DefaultContainerFactoryTests.java

```diff
@@ -1421,6 +1421,22 @@ public void testCommandLineArgsOverridesExistingProperties() {
 		assertThat(defaultContainerFactory.createCommandArgs(appDeploymentRequest)).containsExactly("--foo=newValue");
 	}
 
+	@Test
+	public void testCommandLineArgsNoAssignment() {
+		List<String> args = new ArrayList();
+		args.add("a");
+		args.add("b = c");
```

Comment:
Good test to add. 

Should we also beef up the "override from app definition" testcase to verify that the `b = c` case is handled properly? Its checking on app keys and now we have spaces etc.. (ie "b " != "b")

Reply:
Sounds good.